### ISSUE TITLE
dsorg: PS-E is treated as PDS

### ISSUE THREAD

**Issue by VitekVlcek-Broadcom** [zowe/explorer-mvs#92]
When opening a dataset that has `dsorg: PS-E` explorer consider it as PDS vs sequential dataset, As a result of it does not display content of dataset.

See logs from dataset-api

1) MVS explorer requests information about a dataset via  
```
GET /zosmf/restfiles/ds?dslevel=TSARB.ZOSMFRAC
```
Response:
```json
 {
		"items": [
			{
				"dsname": "TSARB.ZOSMFRAC",
				"blksz": "6233",
				"catnm": "UCAT.HDPLEXT0.TSO",
				"cdate": "2014/07/29",
				"dev": "3390",
				"dsorg": "PS-E",
				"edate": "***None***",
				"extx": "1",
				"lrecl": "256",
				"migr": "NO",
				"mvol": "Y",
				"ovf": "NO",
				"rdate": "2019/10/08",
				"recfm": "VB",
				"sizex": "15",
				"spacu": "CYLINDERS",
				"used": "6",
				"vol": "TSTT00",
				"vols": "TSTT00,*     ,*     ,*     *"
			}
		],
		"returnedRows": 1,
		"JSONversion": 1
	}
```
2) Then `mvs explorer` performs:
```json
GET /zosmf/restfiles/ds/TSARB.ZOSMFRAC/member`
```
Response
```
 		 {
			"category": 4,
			"rc": 12,
			"reason": 0,
			"message": "LMMLIST error",
			"stack": "  1 2302CA48+000000EE IspfInterface::throwIspfException(int,const char*,int)\n  2 22FFA228+0000025E ListMembersServlet::lmmlist()\n  3 22FF6428+00000694 ListMembersServlet::doGet()\n  4 22FFD130+00000086 TsoServlet::service()\n  5 230270D8+0000017E TsoServlet::run()\n  6 23026AB0+00000386 main\n  7 0DDBFAA8+0000127E CEEVROND\n  8 227F4038+000000B4 EDCZHINV\n  9 0DC57928+000001C6 CEEBBEXT\n",
			"details": [
				"ISRZ002 Data set not partitioned - LMMLIST requires the data set to be a partitioned data set."
			]
		}
```

Calling `/zosmf/restfiles/ds/TSARB.ZOSMFRAC/member` is wrong because `TSARB.ZOSMFRAC` is a sequntial dataset, therefore request to obtain content should be used instead.

**Comment by NakulManchanda** [comment]
thank you,
tip from @John-A-Davies 
It's confusing; dataset DSORGs can be
PS Sequential
PS-E Sequential Extended Format
PS-L Large Format Sequential
PSU Sequential unmovable
PO Partitioned
POU Partitioned unmovable
PO-E Partitioned extended
The ones with 'PS*' are sequential, i.e. flat files, and those with 'PO*' are libraries that contain members.

**Comment by jordanCain** [comment]
Closing as resolved by #94
